<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPRUSARW1003" name="Message page testing" tests="4" failures="1" timestamp="31 May 2018 09:26:04 GMT" time="49.96" errors="0">
  <testcase name="" time="5.36" classname="tests.messagePageTests.MessageTest"/>
  <testcase name="" time="3.882" classname="tests.messagePageTests.MessageTest"/>
  <testcase name="" time="33.513" classname="tests.messagePageTests.MessageTest"/>
  <testcase name="" time="3.182" classname="tests.messagePageTests.MessageTest">
    <failure type="java.lang.AssertionError" message="Message with invalid text was sent expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Message with invalid text was sent expected [false] but found [true]
	at tests.messagePageTests.MessageTest.sentMessageWithInvalidText(MessageTest.java:43)
	at runner.Runner.run(Runner.java:49)
	at runner.Runner.main(Runner.java:22)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!--  -->
</testsuite> <!-- Message page testing -->
